analyses/metrics/determine_qc_metric_schema.ipynb

```diff
@@ -0,0 +1,695 @@
+      "reads_mapped_intronic\n",
+      "reads_mapped_utr\n",
+      "reads_mapped_uniquely\n",
+      "reads_mapped_multiple\n",
+      "duplicate_reads\n",
+      "spliced_reads\n",
+      "antisense_reads\n",
+      "molecule_barcode_fraction_bases_above_30_mean\n",
+      "molecule_barcode_fraction_bases_above_30_variance\n",
+      "genomic_reads_fraction_bases_quality_above_30_mean\n",
+      "genomic_reads_fraction_bases_quality_above_30_variance\n",
+      "genomic_read_quality_mean\n",
+      "genomic_read_quality_variance\n",
+      "n_molecules\n",
+      "n_fragments\n",
+      "reads_per_molecule\n",
+      "reads_per_fragment\n",
+      "fragments_per_molecule\n",
+      "fragments_with_single_read_evidence\n",
+      "molecules_with_single_read_evidence\n",
+      "number_cells_detected_multiple\n",
+      "number_cells_expressing\n"
+     ]
+    }
+   ],
+   "source": [
+    "import pandas as pd\n",
+    "\n",
+    "optimus_metric_dir = os.path.expanduser('~/Desktop/optimus')\n",
+    "\n",
+    "print('cell metrics\\n')\n",
+    "for c in pd.read_csv(os.path.join(optimus_metric_dir, 'merged-cell-metrics.csv.gz')).columns[1:]:\n",
+    "    print(c)\n",
+    "\n",
+    "print('\\ngene metrics\\n')\n",
+    "for c in pd.read_csv(os.path.join(optimus_metric_dir, 'merged-gene-metrics.csv.gz')).columns[1:]:\n",
+    "    print(c)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Schema\n",
+    "\n",
+    "Picard does not appear to follow a consistent schema, although one can map their results into JSON using Crimson, which is helpful. The matrix service will require the data be shipped in the same storage format recommend for sequencing data. Currently this appears to be Zarr. \n",
+    "\n",
+    "## ACs for ticket humancellatlas/secondary-analysis#105: \n",
+    "1. List of metrics to be updated: \n",
+    "    1. All, both pipelines. This is a small epic that will span multiple tickets to implement in a forward-compatible fashion.\n",
+    "2. names to be changed to:\n",
+    "    2. N/A, although some of the picard metrics will need to change by nature of their specificity to the files that generated them. For example, \"REF_BASE\" and \"ALT_BASE\" lose substantial meaning when combined into a columnar file. \n",
+    "3. conclusion on this effort: 1.5\n",
+    "\n",
+    "\n",
+    "## Further Implementation Suggestions: \n",
+    "1. Approach Crimson developer about willingness to receive PRs that expand his tool to encompass additional formats (as we receive them). \n",
+    "  1. Is BSD format OK for HCA? Ask about licensing if no. \n",
+    "2. Expand it to read any other metrics. \n",
+    "3. Write some kind of json-metric-to-zarr converter. Keep the 'to-zarr' part separated, in case we need to change things around later. \n",
+    "4. In the process of doing this, determine our own internal schema and a series of maps from other tools' QC metrics. \n",
```

Comment:
+1 to having a standard naming convention for common QC metrics from different tools